documentation/src/docs/getting-started.md

```diff
@@ -36,31 +36,30 @@ $ tar xfvz pravega-<version>.tgz
 
 This launches all the components of Pravega on your local machine.
 
-**Note:** This is for testing/demo purposes only, *do not* use this mode of deployment 
-in Production! 
+**Note:** This is for testing/demo purposes only, **do not** use this mode of deployment
+in Production!
 
 More options to run Pravega can be found in [Running Pravega](deployment/deployment.md) guide.
```

Comment:
This line is actually repeated in line 51 below. Please remove the duplication.

Reply:
@fpj Accepted Suggestion